transform-plugins/README.rst

```diff
@@ -66,18 +60,17 @@ CSV Parser
   Batch and
   Realtime
 :Description:
-  Parses an input field as CSV Record into a Structured Record. Support multi-line CSV record parsing 
-  into multiple structured records. Different formats of CSV record can be parsed using this plugin. 
-  Following are different CSV record types that are supported by this plugin: DEFAULT, EXCEL, MYSQL, RFC4180 and TDF.
+  Parses an input field as a CSV Record into a Structured Record. Supports multi-line CSV Record parsing
+  into multiple Structured Records. Different formats of CSV Record can be parsed using this plugin.
```

Comment:
"Different formats of CSV Record can be parsed using this plugin" is redundant when read with next sentence, right?